demos/python_demos/facies_classification_demo/README.md

```diff
@@ -0,0 +1,46 @@
+# Facies classification Python* Demo
+
+This demo demonstrate how to run facies classification using OpenVINO&trade;
```

Comment:
Do you mind to add more details what is Facies Classification? I mean add few words about the problem, from which domain this model, data origin.